pydis_site/apps/resources/resources/python_discord_videos.yaml

```
@@ -1,3 +1,4 @@
+name: Python Discord YouTube channel
```

It's a title, right? Then this needs to be title case
```suggestion
name: Python Discord YouTube Channel
```